packages/grafana-ui/src/components/Graph/Graph.tsx

```diff
@@ -335,7 +338,7 @@ export class Graph extends PureComponent<GraphProps, GraphState> {
         color: '#a1a1a1',
         margin: { left: 0, right: 0 },
         labelMarginX: 0,
-        mouseActiveRadius: 30,
+        mouseActiveRadius: 15,
```

Comment:
Mouse active radius was changed as at 30 it was a bit difficult to get the desired tooltip when hovering with both bar and line graphs visible